molgenis-questionnaires/src/main/frontend/src/components/ChapterForm.vue

```diff
@@ -10,16 +10,18 @@
 </template>
 
 <style>
-  .questionnaire-chapter .pl-2 > fieldset > small {
+  .questionnaire-chapter .pl-2 > fieldset > small,
+  .questionnaire-chapter .pl-4 > fieldset > legend {
     font-size: 100%;
   }
 
   .questionnaire-chapter [class^="pl-"] > fieldset > legend {
     padding-top: 1rem;
   }
 
+  .questionnaire-chapter [class^="pl-"] > fieldset .form-group > label,
   .questionnaire-chapter .pl-4 > fieldset > legend {
-    font-size: 100%;
+    font-weight: 700;
```

Comment:
Personally i do not like the bold questions, i think it is less readable and we should stick to be bootstrap style.  But that more a matter of option.

Reply:
Ok, I'll remove it and add it to the customer's bootstrap theme then, they really want it.